packages/cli/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @arethetypeswrong/cli
 
+## 0.2.1
+
+### Patch Changes
+
+- Updated dependencies [7c3a377]
+  - @arethetypeswrong/core@0.3.0
```

Comment:
I think I want to make `@arethetypeswrong/cli` [linked](https://github.com/changesets/changesets/blob/main/docs/linked-packages.md) to `@arethetypeswrong/core`. It feels like a major or minor bump of core should necessitate the same in the CLI, but not vice versa. I think I need to add

```json
{
  "linked": [["@arethetypeswrong/cli", "@arethetypeswrong/core"]]
}
```

to my config. @Andarist does that seem right to you?

Reply:
Yeah, that seemed to do the trick. Cool!

Reply:
> Yeah, that seemed to do the trick. Cool!

This could be luck. Those rules are at times tricky so it's worth rechecking the source code and the actual observed behavior. I think that (and [the source code](https://github.com/changesets/changesets/blob/8ede784e5fc3629858a57643fdaab76ffb631520/packages/assemble-release-plan/src/apply-links.ts) roughly seems to confirm that) that this merely ensures that *when* packages are scheduled for a release *then* the highest version within the linked group is assigned to them.

That means that:
- if you release A and later you release B the version assigned to B might be higher than the one assigned to it through changesets (like, imagine that A was previously released with a minor bump and now B got a patch changeset for it)
- this also means that a release of A doesn't automatically include B in the release

Some examples of that behavior might be found [here](https://github.com/changesets/changesets/blob/8ede784e5fc3629858a57643fdaab76ffb631520/docs/linked-packages.md). I'm using this setting in one of the monorepos and it works fine but that whole monorepo kinda operates within a single major release line. So I don't exactly deal with situations in which a major bump to some package should also include another package in that release. 

Overall, Changesets could have more configuration for this kind of stuff... but this is really quite a complex topic with a lot of edge cases. Some manual supervision is advised here. Speaking from purely "technical" PoV... as long as your package is just a dependency of another package then there is no strong reason to transitively create releases for dependants etc. That assumes that the dependency didn't actually break the API contract in a meaningful way.

All in all, this is quite subtle - the "safest" approach is to use fixed packages but that just pulls *everything* into the release each time, and often people want to minimize the number of releases. Especially if a release looks "empty" and doesn't bring any meaningful changes beyond just bumping some ranges in its `package.json`.